pkg/monitor/monitor_cmd/command.go

```diff
@@ -165,8 +196,10 @@ type Timeline struct {
 	MonitorEventFilename string
 	PodResourceFilename  string
 
-	LocatorMatcher map[string][]string
-	Namespaces     []string
+	LocatorMatcher        map[string][]*regexp.Regexp
+	RemovedLocatorMatcher map[string][]*regexp.Regexp
+	Namespaces            []string
+	EndDate               string
```

Comment:
this should be a *time.Time with nil meaning the default

Reply:
hopefully I did that right using `time.Time{}` to denote nil and `IsZero()` to check for nil.

Reply:
>hopefully I did that right using time.Time{} to denote nil and IsZero() to check for nil.

Why not use a pointer?

Reply:
ha, I didn't see the `*` there earlier; I changed it to be `*time.Time`.